app/routes/public.js

```diff
@@ -5,7 +5,7 @@ module.exports = [{
   path: '/robots.txt',
   options: {
     handler: {
-      file: 'app/public/static/robots.txt'
```

Comment:
Should the file be renamed (?app/routes/static.js) or even split into a static and assets versions

Reply:
Thanks both, I've renamed public to static.js